github/resource_github_repository_ruleset_test.go

```diff
@@ -312,6 +332,16 @@ func TestGithubRepositoryRulesets(t *testing.T) {
 						require_last_push_approval        = true
 					}
 
+					aerge_queue {
```

Comment:
I'm not familiar with golang and provider codes.
But I think this is the typo of `merge_queue` considering from test name, isn't it?

Reply:
Good catch, thanks.

Reply:
Thank you for your quick fix.
I really want to use this great feature!